poc/vdaf_pine.py

```diff
@@ -754,6 +767,7 @@ def __init__(self,
             chunk_length_norm_equality=chunk_length_norm_equality,
             num_shares=num_shares,
             num_proofs=2,
+            num_proofs_norm_equality=2,
```

Comment:
What's the motivation for this change? `1` may be a reasonable default given that this part isn't subject to offline attacks.

Reply:
That would be better.